packages/command-palette/src/commands/use-single-site-commands.tsx

```diff
@@ -920,6 +927,10 @@ const useSingleSiteCommands = ( { navigate, currentRoute }: useCommandsParams ):
 				return false;
 			}
 
+			if ( command?.publicOnly && ( isPrivate || isComingSoon ) ) {
```

Comment:
Maybe we could have an `isPublicOnly` property in the command palette config that is:
isPublicOnly => isPrivate || isCommingSoon

That way we could simplify the condition a bit instead of propagating 2 properties wdyt?

Reply:
Yeah I thought about that too. I don't feel strongly but I opted for propagating two props as it is slightly more granular and it's slower to change `jetpack-mu-wpcom` because of how it is deployed to atomic.